CRM/MembershipExtras/Service/MembershipInstalmentsSchedule.php

```diff
@@ -265,10 +325,23 @@ private function getInstalmentsTotalAmount(array $instalments) {
   }
 
   /**
-   * Sets Non Membership Price Field Values
+   * Sets Non Membership Price Field values to the class property.
+   *
+   * @param array $nonMembershipPriceFieldValues
    */
   public function setNonMembershipPriceFieldValues(array $nonMembershipPriceFieldValues) {
     $this->nonMembershipPriceFieldValues = $nonMembershipPriceFieldValues;
   }
 
+  /**
+   * Calculates Single Instalment Amount
+   *
+   * @param $amount
+   * @param $divisor
+   * @return float|int
+   */
+  private function calculateSingleInstalmentAmount($amount, $divisor) {
+    return $amount / $divisor;
```

Comment:
Should we call `round` here with `precision = 2` by default ?

Reply:
@ahed-compucorp  I did not round up there because I want to leave the amount as it is so we will only round up when we need it as it see that the round up function getting duplicated everywhere.